blink/syscall.c

```diff
@@ -1501,6 +1538,12 @@ static int IoctlTcsets(struct Machine *m, int fd, int request, i64 addr,
 static int IoctlTiocgpgrp(struct Machine *m, int fd, i64 addr) {
   int rc;
   u8 *pgrp;
+
+#ifdef __EMSCRIPTEN__
+  // Force shells to disable job control in emscripten
+  return -1;
```

Comment:
Be sure to set `errno`. The documented errnos for `tcgetpgrp` are `ENOTTY` and `EBADF`. I'd recommend using `ENOTTY`.

Reply:
Done